app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapController.java

```diff
@@ -33,31 +33,24 @@
 public class ExploreMapController extends MapController {
 
     private final ExploreMapCalls exploreMapCalls;
-    public LatLng latestSearchLocation; // Can be current and camera target on search this area button is used
-    public LatLng currentLocation; // current location of user
-    public double latestSearchRadius = 0; // Any last search radius
-    public double currentLocationSearchRadius = 0; // Search radius of only searches around current location
-
+    public LatLng latestSearchLocation;         // Last search center
+    public LatLng currentLocation;              // User’s current location
+    public double latestSearchRadius = 0;       // Last search radius
+    public double currentLocationSearchRadius = 0; // Radius when searching around current location
```

Comment:
Would you mind including only changes that are relevant to the issue at hand? Thanks!